link/src/main/java/com/stripe/android/link/injection/LinkPaymentLauncherFactory.kt

```diff
@@ -11,6 +12,7 @@ interface LinkPaymentLauncherFactory {
     fun create(
         @Assisted(MERCHANT_NAME) merchantName: String,
         @Assisted(CUSTOMER_EMAIL) customerEmail: String?,
-        @Assisted(CUSTOMER_PHONE) customerPhone: String?
+        @Assisted(CUSTOMER_PHONE) customerPhone: String?,
+        @Assisted(INITIAL_VALUES_MAP) initialValuesMap: Map<IdentifierSpec, String?>?
```

Comment:
Can you make the names more descriptive? It's not clear what's in the map. Something like INITIAL_SHIPPING_ADDRESS_VALUES_MAP, initialShippingAddressValuesMap is clearer.

Reply:
Or if you're thinking of making this generic (not only for shipping address), maybe `initialFormValuesMap`

Reply:
Sure, I can make it less generic